src/main/java/org/radarcns/util/CSVMessageBodyWriter.java

```diff
@@ -0,0 +1,66 @@
+package org.radarcns.util;
```

Comment:
perhaps move this to `org.radarcns.webapp.media`? Preferably put the package definition below the copyright header, this way the copyright header is automatically collapsed on loading the file.